prbt_hardware_support/include/prbt_hardware_support/modbus_api_spec.h

```diff
@@ -116,6 +118,20 @@ class ModbusApiSpecTemplated
     }
   }
 
+  inline std::pair<unsigned int, unsigned int> getMinMaxRegisterDefintion() const
```

Comment:
I would like to say that a comment is missing but somehow I feel that it rather a missing concept of something like "range". Maybe a dedicated class for more expressiveness?

Reply:
Or... just provide a getMinRegister() and a getMaxRegister() to KISS

Reply:
Why not just getMinRegister() does "Definition" provide any more useful information?

Reply:
It is consistent with the other methods: `setRegisterDefinition()` and so on.

Reply:
Did you choose it, because "setRegister()" would have been misleading?

Reply:
Yep. But also not ideal